.travis.yml

```diff
@@ -1,16 +1,15 @@
 language: node_js
-
 node_js:
-  - '6'
+  - '10'
 
 before_script:
-  - npm install
+  - yarn install
```

Comment:
I'm switching to yarn because that's what looks like this thing is meant to be build with, so the build should be using yarn as well to be consistent.